python/recipe.cpp

```diff
@@ -126,31 +148,24 @@ void register_recipe(pybind11::module& m) {
     using namespace pybind11::literals;
 
     // Connections
-    pybind11::class_<arb::cell_connection> cell_connection(m, "cell_connection",
+    pybind11::class_<cell_connection_shim> cell_connection(m, "cell_connection",
```

Comment:
We need a method for converting a `pyarb::cell_connection_shim` to a `arb::cell_connection`, otherwise pybind11 will have runtime errors when calling converting connections provided in user-defined `connections_on`.